### ISSUE TITLE
install firmware

### ISSUE THREAD

**Issue by MATTAM1981** [makerbase-mks/MKS-EAGLE#6]
Hi, I have some problems installing the firmware, if I insert the sd card with the .bin file nothing happens, I have tried various formatting of the sd card but the result does not change. The screen, mks h43, stays on the home page and nothing happens, I have already updated the screen and it works, do you have any suggestions?

**Comment by Voxel3DPrinting** [comment]
Hi, 
I experienced the same issues.
You need to put the firmware file called "mks_eagle.bin" on the SD card. Not the normal "firmware.bin" as with other mainboards

**Comment by MATTAM1981** [comment]
Yes I know, I have read the guide and followed but the card shows no signs, I also tried to follow the guide for the nano v3, but that doesn't work either
